api/core/rag/datasource/vdb/tidb_vector/tidb_vector.py

```diff
@@ -104,14 +105,14 @@ def _create_collection(self, dimension: int):
                         text TEXT NOT NULL,
                         meta JSON NOT NULL,
                         doc_id VARCHAR(64) AS (JSON_UNQUOTE(JSON_EXTRACT(meta, '$.doc_id'))) STORED,
-                        KEY (doc_id),
                         vector VECTOR<FLOAT>({dimension}) NOT NULL COMMENT "hnsw(distance={self._distance_func})",
```

Comment:
The column comment part can be removed due to the `VECTOR INDEX idx_vector` part

Reply:
Removed.